phing/tasks/properties.xml

```diff
@@ -1,5 +1,14 @@
 <project name="properties" default="build">
 
+  <!-- Set the memory limit for php. -->
+  <if>
+    <isset property="php.mem_limit"/>
```

Comment:
Small request, but I'd prefer to have this variable named memory_limit so that it matches the setting in php.ini.

Also, we already have a line in build.xml that modifies the memory_limit:
https://github.com/acquia/blt/blob/8.x/phing/build.xml#L21

We should modify the existing line rather than adding this to a second place in the code base. This should be simple to accomplish:

```
<property name="php.memory_limit" value="1G" override="false" />
<php expression="ini_set('memory_limit', '${php.memory_limit}');" verbose="true" />
```